tests/unit/test_micrometer_metrics.py

```diff
@@ -48,17 +48,31 @@ def setUp(self) -> None:
             return_value=True,
         ).start()
 
-    def test_paidapps_metrics_registry(self):
+    @patch("buildpack.telemetry.datadog.is_enabled", return_value=False)
+    def test_paidapps_metrics_registry(self, is_enabled):
         with patch.dict(os.environ, {"PROFILE": "some-random-mx-profile"}):
-            result = metrics.configure_influx_registry(Mock())
+            result = metrics.configure_metrics_registry(Mock())
+            metrics_registries = result.get("Metrics.Registries")
             self.assertEqual(
-                result.get("Metrics.Registries"),
+                metrics_registries,
+                metrics.PAIDAPPS_METRICS_REGISTRY,
+            )
+            self.assertEqual(len(metrics_registries), 1)
+
+    @patch("buildpack.telemetry.datadog.is_enabled", return_value=True)
+    def test_paidapps_metrics_registry_statsd(self, is_enabled):
+        with patch.dict(os.environ, {"PROFILE": "some-random-mx-profile"}):
+            result = metrics.configure_metrics_registry(Mock())
+            metrics_registries = result.get("Metrics.Registries")
+            self.assertEqual(
+                metrics_registries,
                 metrics.PAIDAPPS_METRICS_REGISTRY,
```

Comment:
Won't the test for StatsD registry have the new `STATSD_REGISTRY` too?

Reply:
Discussed, test case changed